File: Baekjoon/Python/Implementation/Q1475.py
# 방 번호 / 구현
import sys
import math 
n = list(map(int, sys.stdin.readline().strip()))
cnt = [0] * 9

for k in n:
    print(type(k))
    if k == 6 or k == 9:
        cnt[6] += 0.5
    else:
        cnt[k] += 1
print(math.ceil(max(cnt)))

File: Baekjoon/Swift/Implementation/Q1475.swift
//
//  Q1475.swift
//  BOJ_PS
//
//  Created by 김태훈 on 2021/05/29.
//  방 번호 / 구현

import Foundation

let nums = Array(readLine()!).map { Int(String($0))! }
var res = nums.count
var cnt = [Double](repeating: 0, count: 9)

for n in nums {
    if n == 6 || n == 9 {
        cnt[6] += 0.5
    } else {
        cnt[n] += 1
    }
}

print(Int(round(cnt.max()!)))
